src/adapters/sharethrough.js

```diff
@@ -39,10 +45,9 @@ var SharethroughAdapter = function SharethroughAdapter() {
     return url;
   }
 
-  $$PREBID_GLOBAL$$.strcallback = function(bidResponse) {
+  function _strcallback(bidObj, bidResponse) {
     bidResponse = JSON.parse(bidResponse);
```

Comment:
I see there was another change (causing a conflict) where you wrapped this in a separate try/catch.  I'd still recommend wrapping it in the try block below so that the bid is added to the bidManager (in your `_handleInvalidBid()` function) so it can still be tracked.